plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfiguration.java

```diff
@@ -14,14 +14,30 @@
 import com.google.inject.name.Named;
 
 import org.eclipse.che.api.core.util.SystemInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_CERT_PATH_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_HOST_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_TLS_VERIFY_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.DEFAULT_BOOT2DOCKER_CERTS_DIR;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.DEFAULT_BOOT2DOCKER_URI;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.UNIX_SOCKET_URI;
 
 /**
  * @author Alexander Garagatyi
  */
 public class DockerConnectorConfiguration {
+
+    /**
+     * Logger.
```

Comment:
If it's final and static, it should be in uppercase as well
LOGGER and not logger to be compliant.

About comments, you may dislike it but I'm conforming to code standard http://www.oracle.com/technetwork/articles/java/index-137868.html.